### DISCUSSION TITLE
Is it possible to find symbols beyond workspace?

### DISCUSSION THREAD

**Discussion by maestro4** [microsoft/pylance-release#1193] [Q&A] [answered]
As often development happens using external third party libraries. Currently, I can just find symbols in my workspace but not in those external librariers (that are installed in the conda env).

Is it somehow possible to find symbols in such libraries? Also, I'm wondering whether ctags works with Pylance? It seems to be not working on my development machine.

**Comment by jakebailey** [comment, marked as answer]
I think https://github.com/microsoft/pylance-release/issues/35 is what you're looking for. Feel free to vote on it if you're interested in the feature.

**Reply by maestro4** [reply]
Thanks Jake, exactly what I'm looking for. Unfortunately, the feature is not yet implemented and it seems like it will take quite a bit till it is ready.
